<div class='alert alert-warning'>

SciPy's interactive examples with Jupyterlite are experimental and may not always work as expected. Execution of cells containing imports may result in large downloads (up to 60MB of content for the first import from SciPy). Load times when importing from SciPy may take roughly 10-20 seconds. If you notice any problems, feel free to open an [issue](https://github.com/scipy/scipy/issues/new/choose).

</div>

In epidemiology, individuals are classified as "exposed" or
"unexposed" to some factor or treatment. If the occurrence of some
illness is under study, those who have the illness are often
classified as "cases", and those without it are "noncases".  The
counts of the occurrences of these classes gives a contingency
table
```

exposed    unexposed
cases          a           b
noncases       c           d

```
The sample odds ratio may be written ``(a/c) / (b/d)``.  ``a/c`` can
be interpreted as the odds of a case occurring in the exposed group,
and ``b/d`` as the odds of a case occurring in the unexposed group.
The sample odds ratio is the ratio of these odds.  If the odds ratio
is greater than 1, it suggests that there is a positive association
between being exposed and being a case.

Interchanging the rows or columns of the contingency table inverts
the odds ratio, so it is important to understand the meaning of labels
given to the rows and columns of the table when interpreting the
odds ratio.

Consider a hypothetical example where it is hypothesized that exposure to a
certain chemical is associated with increased occurrence of a certain
disease. Suppose we have the following table for a collection of 410 people
```

exposed unexposed
cases        7       15
noncases    58      472

```
The question we ask is "Is exposure to the chemical associated with
increased risk of the disease?"

Compute the odds ratio:


In [ ]:
from scipy.stats.contingency import odds_ratio
res = odds_ratio([[7, 15], [58, 472]])
res.statistic

3.7836687705553493

For this sample, the odds of getting the disease for those who have been
exposed to the chemical are almost 3.8 times that of those who have not been
exposed.

We can compute the 95% confidence interval for the odds ratio:


In [ ]:
res.confidence_interval(confidence_level=0.95)

ConfidenceInterval(low=1.2514829132266785, high=10.363493716701269)

The 95% confidence interval for the conditional odds ratio is approximately
(1.25, 10.4).

For a more detailed example, see :ref:`hypothesis_odds_ratio`.